src/client/components/table-of-contents/readme.md

```diff
@@ -1 +1,13 @@
 # Table of contents
+
+- Lists all sections on a page
+- Links to each section
+- It's only shown on larger screens
```

Comment:
I would actually like the Table of contents to be visible on smaller screens as well, but just in a different presentation. At another project we have the table of contents on the side on larger screens, and collapsed ("Table of contents ▼") above the content, which the user can toggle to view the Table of contents.